x-pack/plugin/blob-cache/preallocate/src/main/java/module-info.java

```diff
@@ -0,0 +1,8 @@
+module org.elasticsearch.blobcache.preallocate {
+    requires org.elasticsearch.base;
+    requires org.elasticsearch.server;
+    requires org.apache.logging.log4j;
+    requires com.sun.jna;
+
+    exports org.elasticsearch.blobcache.preallocate to org.elasticsearch.blobcache;
```

Comment:
Is this module intended to be reused anywhere else in the future? I guess my question is if the module only exports one package, and only to a single other named module, why not just consolidate them?

Reply:
It's in a separate jar so that security manager permissions can be granted solely to that jar, rather than the entire blobcache project.

Reply:
The answer is always the security manager 😉 . Got it. Thanks.